### PR TITLE
Add CVE-2017-5662

### PR DESCRIPTION
**Pull request by msrb**

### CODE REVIEW

**Comment by vasulakkaraju** [comment]
updated per the ref https://securitytracker.com/id/1038334,  https://mvnrepository.com/artifact/org.apache.xmlgraphics/batik-svg-dom/1.9

File: database/java/2017/5662.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    - https://access.redhat.com/errata/RHSA-2017:2546
+    - https://access.redhat.com/errata/RHSA-2017:2547
+    - https://access.redhat.com/errata/RHSA-2018:0319
+    - https://xmlgraphics.apache.org/security.html
+
+affected:
+    - groupId: org.apache.xmlgraphics
+      artifactId: batik-svg-dom
+      version:
+        - "<= 1.0,1.8"
```
Comment (msrb) [inline database/java/2017/5662.yaml]: The whitespace shouldn't be there and the version should be <=greater,lower, like in "all versions equal or lower than 'greater' are affected, all the way down to 'lower' version."

Comment (msrb) [reply database/java/2017/5662.yaml]: https://github.com/victims/victims-cve-db/

Comment (vasulakkaraju) [reply database/java/2017/5662.yaml]: updated @msrb Thanks

**Comment by msrb** [comment]
LGTM, thanks :wink:

**Pull request merged**